### ISSUE TITLE
Full URLs provided as default image URLs are double encoded

### ISSUE THREAD

**Issue by derekjones** [forxer/gravatar#3]
When providing a full URL as the default image, the resulting URL is being double-encoded and creating an invalid Gravatar URL, e.g.::

```php
Gravatar::image('username@example.com', null, 'https://example.com/image.jpg');

// results in:
// http://www.gravatar.com/avatar/5f0efb20de5ecfedbe0bf5e7c12353fe?d=https%253A%252F%252Fexample.com%252Fimage.jpg
```

Note the `https%253A%252F%252Fexample.com%252Fimage.jpg` instead of `https%3A%2F%2Fexample.com%2Fimage.jpg`.

The URL is first being [raw-encoded](https://github.com/forxer/gravatar/blob/master/src/Image.php#L162) and then later [that value is used in `http_build_query()`](https://github.com/forxer/gravatar/blob/master/src/Image.php#L353).

Since `http_build_query()` already URL-encodes data, I believe the use of `rawencodeurl()` is redundant.

**Comment by forxer** [comment]
You are right, thank you for this report. Fixed in version 1.3.1

**Comment by derekjones** [comment]
:+1: thanks for the quick fix
